src/config.ts

```diff
@@ -61,6 +61,8 @@ export interface AppListViewSettings {
   [ListViews.WEBHOOK_LIST]: ListSettings;
   [ListViews.TRANSLATION_ATTRIBUTE_VALUE_LIST]: ListSettings;
   [ListViews.GIFT_CARD_LIST]: ListSettings;
+  [ListViews.ORDER_DETAILS_LIST]: ListSettings;
+  [ListViews.ORDER_DRAFT_DETAILS_LIST]: ListSettings;
```

Comment:
**suggestion**: I would consider naming them `ORDER_DETAILS` and `ORDER_DRAFT_DETAILS` - they are not strictly list views, they just have a datagrid section.